### ISSUE TITLE
What about a Monospaced font in the DebugConsole editor?

### ISSUE THREAD

**Issue by snobu** [projectkudu/kudu#1599]
This is a small change but with high impact in both visuals and efficiency.

### kudu / Kudu.Services.Web / DebugConsole / Default.cshtml ###
```
Line 105: <textarea class="span12 form-control" ... ></textarea>
```
My proposal would be for a new, additional class: "mono" or "monospace", or if one already exists in the stylesheets, reference that in.

Preview:
![monoconsole](https://cloud.githubusercontent.com/assets/6472374/8342831/d8314302-1ad7-11e5-8daf-df2800876a69.png)

I would be happy to take the time and do it if this sounds good.

**Comment by davidebbo** [comment]
Sure, that would probably make sense. Or maybe we should go all out and get a real editor like Ace in there. We really never invested much time in improving it.

**Comment by ahmelsayed** [comment]
If you just add monospace to the text area, make sure to include some other monospace fonts for Linux and Mac users.

**Comment by snobu** [comment]
Why not bring in the editor from Monaco?
http://m.hanselman.com/blog/ARichNewJavaScriptCodeEditorSpreadingToSeveralMicrosoftWebSites.aspx

From my perspective all that's really needed in Kudu's DebugConsole editor is line numbers and monospaced font. If the full Monaco editor turns out to be a little slow (with syntax hl and autocompletion) then maybe we could strip it down to bare necessities.

**Comment by davidebbo** [comment]
Problem is that Monaco is not OSS. So despite it being used in the Azure portal, it's harder for us to use in Kudu since we're OSS.

**Comment by snobu** [comment]
Right. I did not know that. I'll look for something that adds line numbers without pulling in half a meg of JS. Maybe a jQuery plugin since we already load that. OSS of course.

**Comment by ahmelsayed** [comment]
Closed by 03b4cad78e50b5379c00bc1b7b790b3d72b76fef de0d734944760ac6b675fb41b062109b4e8989b4 e3ce7a165d793348f4f4e37c0059f47ddccaacae 84817365995989628a268dfd8a3b79092c9ee76d and 3b8ad5331fe7c6405cd754109e9d384f38f77144
